### ISSUE TITLE
[Core] File_Upload::isValidFileHandler($field) - Investigate on Phan failure (@phan-suppress-next-line)

### ISSUE THREAD

**Issue by laemtl** [aces/Loris#6491]
**master branch**
Investigate why Phan fails and remove the below comments.
 [php/libraries/File_Upload.class.inc:isValidFileHandler($field)](https://github.com/aces/Loris/blob/master/php/libraries/File_Upload.class.inc#L349)

```
function isValidFileHandler($field)
    {
        $class =&$this->fileHandlers[$field]['class'];
        if (is_object($class)) {
            $test =&$class;
        } else if (class_exists($class)) {
            // Phan seems to think this is always a string and fails as a result.
            // FIXME This should be investigated further but for now it's not
            // breaking anything.
            // @phan-suppress-next-line PhanTypeExpectedObjectOrClassName
            $test =new $class;
        }
```
See #5687 for more details

**Comment by lingma** [comment]
https://github.com/phan/phan/issues/3700

A team member of phan consider this case as a phan's bug.

**Comment by laemtl** [comment]
Fixed by #7200
